multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/exception/InvalidAPIException.java

```diff
@@ -0,0 +1,17 @@
+/*
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.sngular.api.generator.plugin.openapi.exception;
+
+public class InvalidAPIException extends RuntimeException {
+
+  private static final String ERROR_MESSAGE = "OpenApi -> There is at least one operation without operationId";
```

Comment:
Does it make sense to have this class duplicated in both openapi and asyncapi? We have a package for common exceptions